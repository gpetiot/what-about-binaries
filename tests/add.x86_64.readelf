ELF Header:
  Magic:   7f 45 4c 46 02 01 01 00 00 00 00 00 00 00 00 00 
  Class:                             ELF64
  Data:                              2's complement, little endian
  Version:                           1 (current)
  OS/ABI:                            UNIX - System V
  ABI Version:                       0
  Type:                              EXEC (Executable file)
  Machine:                           Advanced Micro Devices X86-64
  Version:                           0x1
  Entry point address:               0x400400
  Start of program headers:          64 (bytes into file)
  Start of section headers:          6608 (bytes into file)
  Flags:                             0x0
  Size of this header:               64 (bytes)
  Size of program headers:           56 (bytes)
  Number of program headers:         9
  Size of section headers:           64 (bytes)
  Number of section headers:         30
  Section header string table index: 27

Section Headers:
  [Nr] Name              Type             Address           Offset
       Size              EntSize          Flags  Link  Info  Align
  [ 0]                   NULL             0000000000000000  00000000
       0000000000000000  0000000000000000           0     0     0
  [ 1] .interp           PROGBITS         0000000000400238  00000238
       000000000000001c  0000000000000000   A       0     0     1
  [ 2] .note.ABI-tag     NOTE             0000000000400254  00000254
       0000000000000020  0000000000000000   A       0     0     4
  [ 3] .note.gnu.build-i NOTE             0000000000400274  00000274
       0000000000000024  0000000000000000   A       0     0     4
  [ 4] .gnu.hash         GNU_HASH         0000000000400298  00000298
       000000000000001c  0000000000000000   A       5     0     8
  [ 5] .dynsym           DYNSYM           00000000004002b8  000002b8
       0000000000000048  0000000000000018   A       6     1     8
  [ 6] .dynstr           STRTAB           0000000000400300  00000300
       0000000000000038  0000000000000000   A       0     0     1
  [ 7] .gnu.version      VERSYM           0000000000400338  00000338
       0000000000000006  0000000000000002   A       5     0     2
  [ 8] .gnu.version_r    VERNEED          0000000000400340  00000340
       0000000000000020  0000000000000000   A       6     1     8
  [ 9] .rela.dyn         RELA             0000000000400360  00000360
       0000000000000018  0000000000000018   A       5     0     8
  [10] .rela.plt         RELA             0000000000400378  00000378
       0000000000000030  0000000000000018  AI       5    12     8
  [11] .init             PROGBITS         00000000004003a8  000003a8
       000000000000001a  0000000000000000  AX       0     0     4
  [12] .plt              PROGBITS         00000000004003d0  000003d0
       0000000000000030  0000000000000010  AX       0     0     16
  [13] .text             PROGBITS         0000000000400400  00000400
       0000000000000182  0000000000000000  AX       0     0     16
  [14] .fini             PROGBITS         0000000000400584  00000584
       0000000000000009  0000000000000000  AX       0     0     4
  [15] .rodata           PROGBITS         0000000000400590  00000590
       0000000000000004  0000000000000004  AM       0     0     4
  [16] .eh_frame_hdr     PROGBITS         0000000000400594  00000594
       0000000000000034  0000000000000000   A       0     0     4
  [17] .eh_frame         PROGBITS         00000000004005c8  000005c8
       00000000000000f4  0000000000000000   A       0     0     8
  [18] .init_array       INIT_ARRAY       0000000000600e10  00000e10
       0000000000000008  0000000000000000  WA       0     0     8
  [19] .fini_array       FINI_ARRAY       0000000000600e18  00000e18
       0000000000000008  0000000000000000  WA       0     0     8
  [20] .jcr              PROGBITS         0000000000600e20  00000e20
       0000000000000008  0000000000000000  WA       0     0     8
  [21] .dynamic          DYNAMIC          0000000000600e28  00000e28
       00000000000001d0  0000000000000010  WA       6     0     8
  [22] .got              PROGBITS         0000000000600ff8  00000ff8
       0000000000000008  0000000000000008  WA       0     0     8
  [23] .got.plt          PROGBITS         0000000000601000  00001000
       0000000000000028  0000000000000008  WA       0     0     8
  [24] .data             PROGBITS         0000000000601028  00001028
       0000000000000018  0000000000000000  WA       0     0     8
  [25] .bss              NOBITS           0000000000601040  00001040
       0000000000000008  0000000000000000  WA       0     0     1
  [26] .comment          PROGBITS         0000000000000000  00001040
       000000000000002d  0000000000000001  MS       0     0     1
  [27] .shstrtab         STRTAB           0000000000000000  0000106d
       0000000000000108  0000000000000000           0     0     1
  [28] .symtab           SYMTAB           0000000000000000  00001178
       0000000000000630  0000000000000018          29    45     8
  [29] .strtab           STRTAB           0000000000000000  000017a8
       0000000000000227  0000000000000000           0     0     1
Key to Flags:
  W (write), A (alloc), X (execute), M (merge), S (strings), l (large)
  I (info), L (link order), G (group), T (TLS), E (exclude), x (unknown)
  O (extra OS processing required) o (OS specific), p (processor specific)

Program Headers:
  Type           Offset             VirtAddr           PhysAddr
                 FileSiz            MemSiz              Flags  Align
  PHDR           0x0000000000000040 0x0000000000400040 0x0000000000400040
                 0x00000000000001f8 0x00000000000001f8  R E    8
  INTERP         0x0000000000000238 0x0000000000400238 0x0000000000400238
                 0x000000000000001c 0x000000000000001c  R      1
      [Requesting program interpreter: /lib64/ld-linux-x86-64.so.2]
  LOAD           0x0000000000000000 0x0000000000400000 0x0000000000400000
                 0x00000000000006bc 0x00000000000006bc  R E    200000
  LOAD           0x0000000000000e10 0x0000000000600e10 0x0000000000600e10
                 0x0000000000000230 0x0000000000000238  RW     200000
  DYNAMIC        0x0000000000000e28 0x0000000000600e28 0x0000000000600e28
                 0x00000000000001d0 0x00000000000001d0  RW     8
  NOTE           0x0000000000000254 0x0000000000400254 0x0000000000400254
                 0x0000000000000044 0x0000000000000044  R      4
  GNU_EH_FRAME   0x0000000000000594 0x0000000000400594 0x0000000000400594
                 0x0000000000000034 0x0000000000000034  R      4
  GNU_STACK      0x0000000000000000 0x0000000000000000 0x0000000000000000
                 0x0000000000000000 0x0000000000000000  RW     10
  GNU_RELRO      0x0000000000000e10 0x0000000000600e10 0x0000000000600e10
                 0x00000000000001f0 0x00000000000001f0  R      1

 Section to Segment mapping:
  Segment Sections...
   00     
   01     .interp 
   02     .interp .note.ABI-tag .note.gnu.build-id .gnu.hash .dynsym .dynstr .gnu.version .gnu.version_r .rela.dyn .rela.plt .init .plt .text .fini .rodata .eh_frame_hdr .eh_frame 
   03     .init_array .fini_array .jcr .dynamic .got .got.plt .data .bss 
   04     .dynamic 
   05     .note.ABI-tag .note.gnu.build-id 
   06     .eh_frame_hdr 
   07     
   08     .init_array .fini_array .jcr .dynamic .got 

Symbol table '.dynsym' contains 3 entries:
   Num:    Value          Size Type    Bind   Vis      Ndx Name
     0: 0000000000000000     0 NOTYPE  LOCAL  DEFAULT  UND 
     1: 0000000000000000     0 FUNC    GLOBAL DEFAULT  UND __libc_start_main@GLIBC_2.2.5 (2)
     2: 0000000000000000     0 NOTYPE  WEAK   DEFAULT  UND __gmon_start__

Symbol table '.symtab' contains 66 entries:
   Num:    Value          Size Type    Bind   Vis      Ndx Name
     0: 0000000000000000     0 NOTYPE  LOCAL  DEFAULT  UND 
     1: 0000000000400238     0 SECTION LOCAL  DEFAULT    1 
     2: 0000000000400254     0 SECTION LOCAL  DEFAULT    2 
     3: 0000000000400274     0 SECTION LOCAL  DEFAULT    3 
     4: 0000000000400298     0 SECTION LOCAL  DEFAULT    4 
     5: 00000000004002b8     0 SECTION LOCAL  DEFAULT    5 
     6: 0000000000400300     0 SECTION LOCAL  DEFAULT    6 
     7: 0000000000400338     0 SECTION LOCAL  DEFAULT    7 
     8: 0000000000400340     0 SECTION LOCAL  DEFAULT    8 
     9: 0000000000400360     0 SECTION LOCAL  DEFAULT    9 
    10: 0000000000400378     0 SECTION LOCAL  DEFAULT   10 
    11: 00000000004003a8     0 SECTION LOCAL  DEFAULT   11 
    12: 00000000004003d0     0 SECTION LOCAL  DEFAULT   12 
    13: 0000000000400400     0 SECTION LOCAL  DEFAULT   13 
    14: 0000000000400584     0 SECTION LOCAL  DEFAULT   14 
    15: 0000000000400590     0 SECTION LOCAL  DEFAULT   15 
    16: 0000000000400594     0 SECTION LOCAL  DEFAULT   16 
    17: 00000000004005c8     0 SECTION LOCAL  DEFAULT   17 
    18: 0000000000600e10     0 SECTION LOCAL  DEFAULT   18 
    19: 0000000000600e18     0 SECTION LOCAL  DEFAULT   19 
    20: 0000000000600e20     0 SECTION LOCAL  DEFAULT   20 
    21: 0000000000600e28     0 SECTION LOCAL  DEFAULT   21 
    22: 0000000000600ff8     0 SECTION LOCAL  DEFAULT   22 
    23: 0000000000601000     0 SECTION LOCAL  DEFAULT   23 
    24: 0000000000601028     0 SECTION LOCAL  DEFAULT   24 
    25: 0000000000601040     0 SECTION LOCAL  DEFAULT   25 
    26: 0000000000000000     0 SECTION LOCAL  DEFAULT   26 
    27: 0000000000000000     0 FILE    LOCAL  DEFAULT  ABS crtstuff.c
    28: 0000000000600e20     0 OBJECT  LOCAL  DEFAULT   20 __JCR_LIST__
    29: 0000000000400430     0 FUNC    LOCAL  DEFAULT   13 deregister_tm_clones
    30: 0000000000400470     0 FUNC    LOCAL  DEFAULT   13 register_tm_clones
    31: 00000000004004b0     0 FUNC    LOCAL  DEFAULT   13 __do_global_dtors_aux
    32: 0000000000601040     1 OBJECT  LOCAL  DEFAULT   25 completed.7568
    33: 0000000000600e18     0 OBJECT  LOCAL  DEFAULT   19 __do_global_dtors_aux_fin
    34: 00000000004004d0     0 FUNC    LOCAL  DEFAULT   13 frame_dummy
    35: 0000000000600e10     0 OBJECT  LOCAL  DEFAULT   18 __frame_dummy_init_array_
    36: 0000000000000000     0 FILE    LOCAL  DEFAULT  ABS add.c
    37: 0000000000000000     0 FILE    LOCAL  DEFAULT  ABS crtstuff.c
    38: 00000000004006b8     0 OBJECT  LOCAL  DEFAULT   17 __FRAME_END__
    39: 0000000000600e20     0 OBJECT  LOCAL  DEFAULT   20 __JCR_END__
    40: 0000000000000000     0 FILE    LOCAL  DEFAULT  ABS 
    41: 0000000000600e18     0 NOTYPE  LOCAL  DEFAULT   18 __init_array_end
    42: 0000000000600e28     0 OBJECT  LOCAL  DEFAULT   21 _DYNAMIC
    43: 0000000000600e10     0 NOTYPE  LOCAL  DEFAULT   18 __init_array_start
    44: 0000000000601000     0 OBJECT  LOCAL  DEFAULT   23 _GLOBAL_OFFSET_TABLE_
    45: 0000000000400580     2 FUNC    GLOBAL DEFAULT   13 __libc_csu_fini
    46: 0000000000000000     0 NOTYPE  WEAK   DEFAULT  UND _ITM_deregisterTMCloneTab
    47: 0000000000601028     0 NOTYPE  WEAK   DEFAULT   24 data_start
    48: 0000000000601040     0 NOTYPE  GLOBAL DEFAULT   24 _edata
    49: 0000000000601038     4 OBJECT  GLOBAL DEFAULT   24 x
    50: 0000000000400584     0 FUNC    GLOBAL DEFAULT   14 _fini
    51: 0000000000000000     0 FUNC    GLOBAL DEFAULT  UND __libc_start_main@@GLIBC_
    52: 0000000000601028     0 NOTYPE  GLOBAL DEFAULT   24 __data_start
    53: 0000000000000000     0 NOTYPE  WEAK   DEFAULT  UND __gmon_start__
    54: 0000000000601030     0 OBJECT  GLOBAL HIDDEN    24 __dso_handle
    55: 0000000000400590     4 OBJECT  GLOBAL DEFAULT   15 _IO_stdin_used
    56: 0000000000400510   101 FUNC    GLOBAL DEFAULT   13 __libc_csu_init
    57: 0000000000601048     0 NOTYPE  GLOBAL DEFAULT   25 _end
    58: 0000000000400400    42 FUNC    GLOBAL DEFAULT   13 _start
    59: 000000000060103c     4 OBJECT  GLOBAL DEFAULT   24 y
    60: 0000000000601040     0 NOTYPE  GLOBAL DEFAULT   25 __bss_start
    61: 00000000004004f6    20 FUNC    GLOBAL DEFAULT   13 main
    62: 0000000000000000     0 NOTYPE  WEAK   DEFAULT  UND _Jv_RegisterClasses
    63: 0000000000601040     0 OBJECT  GLOBAL HIDDEN    24 __TMC_END__
    64: 0000000000000000     0 NOTYPE  WEAK   DEFAULT  UND _ITM_registerTMCloneTable
    65: 00000000004003a8     0 FUNC    GLOBAL DEFAULT   11 _init
