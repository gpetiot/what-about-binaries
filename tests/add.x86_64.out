ELF Header:
  Class:                             ELF64
  Data:                              little endian
  Version:                           1
  OS/ABI:                            0
  ABI Version:                       0
  Type:                              EXEC (Executable file)
  Machine:                           X86_64
  Entry point address:               0x400400
  Start of program headers:          64 (bytes into file)
  Start of section headers:          6608 (bytes into file)
  Flags:                             0x0
  Size of this header:               64 (bytes)
  Size of program headers:           56 (bytes)
  Number of program headers:         9
  Size of section headers:           64 (bytes)
  Number of section headers:         30
  Section header string table index: 27

Section Headers:
  [Nr] Name              Type            Addr     Off    Size   ES Flg Lk Inf Al
  [ 0]                   NULL            00000000 00000000 00000000 00   0  0   0  0
  [ 1] .interp           PROGBITS        00400238 00000238 0000001c 00   2  0   0  1
  [ 2] .note.ABI-tag     NOTE            00400254 00000254 00000020 00   2  0   0  4
  [ 3] .note.gnu.build-id NOTE            00400274 00000274 00000024 00   2  0   0  4
  [ 4] .gnu.hash         GNU_HASH        00400298 00000298 0000001c 00   2  5   0  8
  [ 5] .dynsym           DYNSYM          004002b8 000002b8 00000048 18   2  6   1  8
  [ 6] .dynstr           STRTAB          00400300 00000300 00000038 00   2  0   0  1
  [ 7] .gnu.version      VERSYM          00400338 00000338 00000006 02   2  5   0  2
  [ 8] .gnu.version_r    VERNEED         00400340 00000340 00000020 00   2  6   1  8
  [ 9] .rela.dyn         RELA            00400360 00000360 00000018 18   2  5   0  8
  [10] .rela.plt         RELA            00400378 00000378 00000030 18  66  5  12  8
  [11] .init             PROGBITS        004003a8 000003a8 0000001a 00   6  0   0  4
  [12] .plt              PROGBITS        004003d0 000003d0 00000030 10   6  0   0 16
  [13] .text             PROGBITS        00400400 00000400 00000182 00   6  0   0 16
  [14] .fini             PROGBITS        00400584 00000584 00000009 00   6  0   0  4
  [15] .rodata           PROGBITS        00400590 00000590 00000004 04  18  0   0  4
  [16] .eh_frame_hdr     PROGBITS        00400594 00000594 00000034 00   2  0   0  4
  [17] .eh_frame         PROGBITS        004005c8 000005c8 000000f4 00   2  0   0  8
  [18] .init_array       INIT_ARRAY      00600e10 00000e10 00000008 00   3  0   0  8
  [19] .fini_array       FINI_ARRAY      00600e18 00000e18 00000008 00   3  0   0  8
  [20] .jcr              PROGBITS        00600e20 00000e20 00000008 00   3  0   0  8
  [21] .dynamic          DYNAMIC         00600e28 00000e28 000001d0 10   3  6   0  8
  [22] .got              PROGBITS        00600ff8 00000ff8 00000008 08   3  0   0  8
  [23] .got.plt          PROGBITS        00601000 00001000 00000028 08   3  0   0  8
  [24] .data             PROGBITS        00601028 00001028 00000018 00   3  0   0  8
  [25] .bss              NOBITS          00601040 00001040 00000008 00   3  0   0  1
  [26] .comment          PROGBITS        00000000 00001040 0000002d 01  48  0   0  1
  [27] .shstrtab         STRTAB          00000000 0000106d 00000108 00   0  0   0  1
  [28] .symtab           SYMTAB          00000000 00001178 00000630 18   0 29  45  8
  [29] .strtab           STRTAB          00000000 000017a8 00000227 00   0  0   0  1
Key to Flags:
  W (write), A (alloc), X (execute), M (merge), S (strings)
  I (info), L (link order), G (group), T (TLS), E (exclude), x (unknown)
  O (extra OS processing required) o (OS specific), p (processor specific)

Program Headers:
  Type           Offset   VirtAddr   PhysAddr   FileSiz MemSiz  Flg Align
  PHDR           0x0000000000000040 0x0000000000400040 0x0000000000400040 0x1f8 0x1f8 R E 8
  INTERP         0x0000000000000238 0x0000000000400238 0x0000000000400238 0x1c 0x1c R   1
  LOAD           0x0000000000000000 0x0000000000400000 0x0000000000400000 0x6bc 0x6bc R E 200000
  LOAD           0x0000000000000e10 0x0000000000600e10 0x0000000000600e10 0x230 0x238 RW  200000
  DYNAMIC        0x0000000000000e28 0x0000000000600e28 0x0000000000600e28 0x1d0 0x1d0 RW  8
  NOTE           0x0000000000000254 0x0000000000400254 0x0000000000400254 0x44 0x44 R   4
  GNU_EH_FRAME   0x0000000000000594 0x0000000000400594 0x0000000000400594 0x34 0x34 R   4
  GNU_STACK      0x0000000000000000 0x0000000000000000 0x0000000000000000 0x0 0x0 RW  10
  GNU_RELRO      0x0000000000000e10 0x0000000000600e10 0x0000000000600e10 0x1f0 0x1f0 R   1

Symbol table '.dynsym' contains 3 entries:
   Num:    Value  Size Type    Bind   Vis      Ndx Name
     0: 00000000     0 NOTYPE  LOCAL  DEFAULT  UND 
     1: 00000000     0 FUNC    GLOBAL DEFAULT  UND __libc_start_main
     2: 00000000     0 NOTYPE  WEAK   DEFAULT  UND __gmon_start__

Symbol table '.symtab' contains 66 entries:
   Num:    Value  Size Type    Bind   Vis      Ndx Name
     0: 00000000     0 NOTYPE  LOCAL  DEFAULT  UND 
     1: 00400238     0 SECTION LOCAL  DEFAULT    1 
     2: 00400254     0 SECTION LOCAL  DEFAULT    2 
     3: 00400274     0 SECTION LOCAL  DEFAULT    3 
     4: 00400298     0 SECTION LOCAL  DEFAULT    4 
     5: 004002b8     0 SECTION LOCAL  DEFAULT    5 
     6: 00400300     0 SECTION LOCAL  DEFAULT    6 
     7: 00400338     0 SECTION LOCAL  DEFAULT    7 
     8: 00400340     0 SECTION LOCAL  DEFAULT    8 
     9: 00400360     0 SECTION LOCAL  DEFAULT    9 
    10: 00400378     0 SECTION LOCAL  DEFAULT   10 
    11: 004003a8     0 SECTION LOCAL  DEFAULT   11 
    12: 004003d0     0 SECTION LOCAL  DEFAULT   12 
    13: 00400400     0 SECTION LOCAL  DEFAULT   13 
    14: 00400584     0 SECTION LOCAL  DEFAULT   14 
    15: 00400590     0 SECTION LOCAL  DEFAULT   15 
    16: 00400594     0 SECTION LOCAL  DEFAULT   16 
    17: 004005c8     0 SECTION LOCAL  DEFAULT   17 
    18: 00600e10     0 SECTION LOCAL  DEFAULT   18 
    19: 00600e18     0 SECTION LOCAL  DEFAULT   19 
    20: 00600e20     0 SECTION LOCAL  DEFAULT   20 
    21: 00600e28     0 SECTION LOCAL  DEFAULT   21 
    22: 00600ff8     0 SECTION LOCAL  DEFAULT   22 
    23: 00601000     0 SECTION LOCAL  DEFAULT   23 
    24: 00601028     0 SECTION LOCAL  DEFAULT   24 
    25: 00601040     0 SECTION LOCAL  DEFAULT   25 
    26: 00000000     0 SECTION LOCAL  DEFAULT   26 
    27: 00000000     0 FILE    LOCAL  DEFAULT  ABS crtstuff.c
    28: 00600e20     0 OBJECT  LOCAL  DEFAULT   20 __JCR_LIST__
    29: 00400430     0 FUNC    LOCAL  DEFAULT   13 deregister_tm_clones
    30: 00400470     0 FUNC    LOCAL  DEFAULT   13 register_tm_clones
    31: 004004b0     0 FUNC    LOCAL  DEFAULT   13 __do_global_dtors_aux
    32: 00601040     1 OBJECT  LOCAL  DEFAULT   25 completed.7568
    33: 00600e18     0 OBJECT  LOCAL  DEFAULT   19 __do_global_dtors_aux_fini_array_entry
    34: 004004d0     0 FUNC    LOCAL  DEFAULT   13 frame_dummy
    35: 00600e10     0 OBJECT  LOCAL  DEFAULT   18 __frame_dummy_init_array_entry
    36: 00000000     0 FILE    LOCAL  DEFAULT  ABS add.c
    37: 00000000     0 FILE    LOCAL  DEFAULT  ABS crtstuff.c
    38: 004006b8     0 OBJECT  LOCAL  DEFAULT   17 __FRAME_END__
    39: 00600e20     0 OBJECT  LOCAL  DEFAULT   20 __JCR_END__
    40: 00000000     0 FILE    LOCAL  DEFAULT  ABS 
    41: 00600e18     0 NOTYPE  LOCAL  DEFAULT   18 __init_array_end
    42: 00600e28     0 OBJECT  LOCAL  DEFAULT   21 _DYNAMIC
    43: 00600e10     0 NOTYPE  LOCAL  DEFAULT   18 __init_array_start
    44: 00601000     0 OBJECT  LOCAL  DEFAULT   23 _GLOBAL_OFFSET_TABLE_
    45: 00400580     2 FUNC    GLOBAL DEFAULT   13 __libc_csu_fini
    46: 00000000     0 NOTYPE  WEAK   DEFAULT  UND _ITM_deregisterTMCloneTable
    47: 00601028     0 NOTYPE  WEAK   DEFAULT   24 data_start
    48: 00601040     0 NOTYPE  GLOBAL DEFAULT   24 _edata
    49: 00601038     4 OBJECT  GLOBAL DEFAULT   24 x
    50: 00400584     0 FUNC    GLOBAL DEFAULT   14 _fini
    51: 00000000     0 FUNC    GLOBAL DEFAULT  UND __libc_start_main@@GLIBC_2.2.5
    52: 00601028     0 NOTYPE  GLOBAL DEFAULT   24 __data_start
    53: 00000000     0 NOTYPE  WEAK   DEFAULT  UND __gmon_start__
    54: 00601030     0 OBJECT  GLOBAL HIDDEN    24 __dso_handle
    55: 00400590     4 OBJECT  GLOBAL DEFAULT   15 _IO_stdin_used
    56: 00400510   101 FUNC    GLOBAL DEFAULT   13 __libc_csu_init
    57: 00601048     0 NOTYPE  GLOBAL DEFAULT   25 _end
    58: 00400400    42 FUNC    GLOBAL DEFAULT   13 _start
    59: 0060103c     4 OBJECT  GLOBAL DEFAULT   24 y
    60: 00601040     0 NOTYPE  GLOBAL DEFAULT   25 __bss_start
    61: 004004f6    20 FUNC    GLOBAL DEFAULT   13 main
    62: 00000000     0 NOTYPE  WEAK   DEFAULT  UND _Jv_RegisterClasses
    63: 00601040     0 OBJECT  GLOBAL HIDDEN    24 __TMC_END__
    64: 00000000     0 NOTYPE  WEAK   DEFAULT  UND _ITM_registerTMCloneTable
    65: 004003a8     0 FUNC    GLOBAL DEFAULT   11 _init

Instructions:
0x1000	push	rbp
0x1001	mov	rbp, rsp
0x1004	mov	edx, dword ptr [rip + 0x200b38]
0x100a	mov	eax, dword ptr [rip + 0x200b36]
0x1010	add	eax, edx
0x1012	pop	rbp
0x1013	ret	
